jcl/src/java.base/share/classes/com/ibm/oti/util/ExternalMessages-MasterIndex.properties

```diff
@@ -1160,6 +1160,7 @@ K0660="Internal error while obtaining private constructor."
 K0661="Internal error while obtaining GcInfo instance."
 K0662="maxDepth must not be negative."
 K0663="Invalid or unsupported dump agent option, cannot be triggered."
+K0664="{0} is not in an exported package"
```

Comment:
Is there no existing message that can be reused?

Reply:
Yes.  Fixed.